chart/env/dev.yaml

```diff
@@ -159,11 +159,11 @@ api:
 
 workers:
   -
-    deployName: "generic"
+    deployName: "all"
     maxJobsPerNamespace: 1
-    workerOnlyJobTypes: ""
+    workerOnlyJobTypes: "/config-names,/split-names-from-streaming,splits,/first-rows,/parquet-and-dataset-info,"
```

Comment:
I would let this empty. It's easier to maintain, and processing light jobs in this worker will have no visible effect (they are processed in less than one second)

Reply:
Done in https://github.com/huggingface/datasets-server/pull/945/commits/47e16e3cdb88b1005c34bec34c181162008fe1d3